packages/web_benchmarks/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: web_benchmarks
 description: A benchmark harness for performance-testing Flutter apps in Chrome.
 repository: https://github.com/flutter/packages/tree/main/packages/web_benchmarks
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+web_benchmarks%22
-version: 0.1.0+9
+version: 0.1.0+10
 
 environment:
   sdk: ">=3.0.0 <4.0.0"
```

Comment:
You'll need to bump these up to match `stable` so the legacy analysis tests don't run and fail.

Reply:
cc @mdebbar Do you recall in which stable Flutter release the deprecation happened?

Reply:
This is the commit: https://github.com/flutter/engine/commit/73671d4ade13e020ef0a86ebc1f961af714658d9. According to the version tags, the earliest stable release that contains that commit was `3.16.0` (I think?).